template_feed/Microsoft.DotNet.Common.ItemTemplates/content/RazorPage/Index.cshtml

```diff
@@ -0,0 +1,6 @@
+﻿@page
+@*#if (!no-pagemodel)*@
+@model Index
+@*#endif*@
+@{
+}
```

Comment:
Don't​we usually include blank lines at the end of template files? I think it makes it easier to start adding HTML, etc.